### PR TITLE
Add leaderboards

### LINKED ISSUE #93: Create multi-site compatible leaderboard
- [x] Select the columns in the leaderboard (@tschaffter )
- [ ] Implement the leaderboard (@thomasyu888 )

### PR DESCRIPTION
**Pull request by thomasyu888**
- Add in leaderboards
- depends on #99

### CODE REVIEW

File: wiki/608039-DateAnnotationLeaderboard.md [hunk 1]
```diff
@@ -2,10 +2,14 @@
 {row}
 {column width=12}
 
+- Latest schema version: `1.1.2`
```
Comment (thomasyu888) [inline wiki/608039-DateAnnotationLeaderboard.md]: This was originally added below the leaderboard widget, but the physical address has two leaderboards, so I added it to the top.  Decision is to create a dedicated section below.

File: wiki/608039-DateAnnotationLeaderboard.md [hunk 2]
```diff
@@ -4,8 +4,14 @@
...
+${synapsetable?query=select id as "Submission Id"%2C createdOn as "CreatedOn"%2C submitterid as Submitter%2C tool%5F%5Fapi%5Fversion as "NLP Sandbox Version"%2C  location%5FF1%5Finstance%5Fstrict as "i2b2 Score"%2C MCW%5Flocation%5FF1%5Finstance%5Fstrict as "MCW score"%2C tool%5F%5Fname as "Tool Name"%2C  tool%5F%5Fversion as "Tool Version"%2C tool%5F%5Furl as "Tool URL"%2C tool%5F%5Fdescription as "Tool Description"%2C tool%5F%5Flicense as "License"%2C dockerrepositoryname as "Image"%2C dockerdigest as "Image Digest"%2C dataset%5Fname as "i2b2 dataset"%2C dataset%5Fversion as "i2b2 dataset version"%2C MCW%5Fdataset%5Fname as "MCW dataset name"%2C MCW%5Fdataset%5Fversion as "MCW dataset version" from  syn23747126 where evaluationid %3D9614652 and status %3D %27ACCEPTED%27&showquery=false}
+
+### Additional Information
+
+- Latest schema version: `1.1.2`
+- Latest version of the i2b2 dataset: `i2b2-phi-20210606`
+- Latest version of the MCW dataset: `mcwdataset-20210525`
+- What the score column is: `F1 instance strict`
```
Comment (tschaffter) [inline wiki/608039-DateAnnotationLeaderboard.md]:
```suggestion
- Score definition: `F1 instance strict`
```

File: wiki/608040-PersonNameAnnotationLeaderboard.md [hunk 3]
```diff
@@ -4,8 +4,14 @@
...
+${synapsetable?query=select id as "Submission Id"%2C createdOn as "CreatedOn"%2C submitterid as Submitter%2C tool%5F%5Fapi%5Fversion as "NLP Sandbox Version"%2C  location%5FF1%5Finstance%5Fstrict as "i2b2 Score"%2C MCW%5Flocation%5FF1%5Finstance%5Fstrict as "MCW score"%2C tool%5F%5Fname as "Tool Name"%2C  tool%5F%5Fversion as "Tool Version"%2C tool%5F%5Furl as "Tool URL"%2C tool%5F%5Fdescription as "Tool Description"%2C tool%5F%5Flicense as "License"%2C dockerrepositoryname as "Image"%2C dockerdigest as "Image Digest"%2C dataset%5Fname as "i2b2 dataset"%2C dataset%5Fversion as "i2b2 dataset version"%2C MCW%5Fdataset%5Fname as "MCW dataset name"%2C MCW%5Fdataset%5Fversion as "MCW dataset version" from  syn23747126 where evaluationid %3D9614657 and status %3D %27ACCEPTED%27&showquery=false}
+
+### Additional Information
+
+- Latest schema version: `1.1.2`
+- Latest version of the i2b2 dataset: `i2b2-phi-20210606`
+- Latest version of the MCW dataset: `mcwdataset-20210525`
+- What the score column is: `F1 instance strict`
```
Comment (tschaffter) [inline wiki/608040-PersonNameAnnotationLeaderboard.md]:
```suggestion
- Score definition: `F1 instance strict`
```

File: wiki/608041-PhysicalAddressAnnotationLeaderboard.md [hunk 4]
```diff
@@ -4,13 +4,18 @@
...
+${synapsetable?query=select id as "Submission Id"%2C createdOn as "CreatedOn"%2C submitterid as Submitter%2C tool%5F%5Fapi%5Fversion as "NLP Sandbox Version"%2C  location%5FF1%5Finstance%5Fstrict as "i2b2 Score"%2C MCW%5Flocation%5FF1%5Finstance%5Fstrict as "MCW score"%2C tool%5F%5Fname as "Tool Name"%2C  tool%5F%5Fversion as "Tool Version"%2C tool%5F%5Furl as "Tool URL"%2C tool%5F%5Fdescription as "Tool Description"%2C tool%5F%5Flicense as "License"%2C dockerrepositoryname as "Image"%2C dockerdigest as "Image Digest"%2C dataset%5Fname as "i2b2 dataset"%2C dataset%5Fversion as "i2b2 dataset version"%2C MCW%5Fdataset%5Fname as "MCW dataset name"%2C MCW%5Fdataset%5Fversion  as "MCW dataset version" from  syn23747126 where evaluationid %3D9614658 and status %3D %27ACCEPTED%27&showquery=false}
+
+### Additional Information
+
+- Latest schema version: `1.1.2`
+- Latest version of the i2b2 dataset: `i2b2-phi-20210606`
+- Latest version of the MCW dataset: `mcwdataset-20210525`
+- What the score column is: `F1 instance strict`
```
Comment (tschaffter) [inline wiki/608041-PhysicalAddressAnnotationLeaderboard.md]:
```suggestion
- Score definition: `F1 instance strict`
```

File: wiki/611107-IdAnnotationLeaderboard.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+### Annotation Location
+
+${synapsetable?query=select id as "Submission Id"%2C createdOn as "CreatedOn"%2C submitterid as Submitter%2C tool%5F%5Fapi%5Fversion as "NLP Sandbox Version"%2C  location%5FF1%5Finstance%5Fstrict as "i2b2 Score"%2C MCW%5Flocation%5FF1%5Finstance%5Fstrict as "MCW score"%2C tool%5F%5Fname as "Tool Name"%2C  tool%5F%5Fversion as "Tool Version"%2C tool%5F%5Furl as "Tool URL"%2C tool%5F%5Fdescription as "Tool Description"%2C tool%5F%5Flicense as "License"%2C dockerrepositoryname as "Image"%2C dockerdigest as "Image Digest"%2C dataset%5Fname as "i2b2 dataset"%2C dataset%5Fversion as "i2b2 dataset version"%2C MCW%5Fdataset%5Fname as "MCW dataset name"%2C MCW%5Fdataset%5Fversion as "MCW dataset version" from  syn23747126 where evaluationid %3D9614803 and status %3D %27ACCEPTED%27&showquery=false}
+
+### Additional Information
+
+- Latest schema version: `1.1.2`
+- Latest version of the i2b2 dataset: `i2b2-phi-20210606`
+- Latest version of the MCW dataset: `mcwdataset-20210525`
+- What the score column is: `F1 instance strict`
```
Comment (tschaffter) [inline wiki/611107-IdAnnotationLeaderboard.md]:
```suggestion
- Score definition: `F1 instance strict`
```

File: wiki/611108-ContactAnnotationLeaderboard.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+### Annotation Location
+
+${synapsetable?query=select id as "Submission Id"%2C createdOn as "CreatedOn"%2C submitterid as Submitter%2C tool%5F%5Fapi%5Fversion as "NLP Sandbox Version"%2C  location%5FF1%5Finstance%5Fstrict as "i2b2 Score"%2C MCW%5Flocation%5FF1%5Finstance%5Fstrict as "MCW score"%2C tool%5F%5Fname as "Tool Name"%2C  tool%5F%5Fversion as "Tool Version"%2C tool%5F%5Furl as "Tool URL"%2C tool%5F%5Fdescription as "Tool Description"%2C tool%5F%5Flicense as "License"%2C dockerrepositoryname as "Image"%2C dockerdigest as "Image Digest"%2C dataset%5Fname as "i2b2 dataset"%2C dataset%5Fversion as "i2b2 dataset version"%2C MCW%5Fdataset%5Fname as "MCW dataset name"%2C MCW%5Fdataset%5Fversion as "MCW dataset version" from  syn23747126 where evaluationid %3D9614799 and status %3D %27ACCEPTED%27&showquery=false}
+
+### Additional Information
+
+- Latest schema version: `1.1.2`
+- Latest version of the i2b2 dataset: `i2b2-phi-20210606`
+- Latest version of the MCW dataset: `mcwdataset-20210525`
+- What the score column is: `F1 instance strict`
```
Comment (tschaffter) [inline wiki/611108-ContactAnnotationLeaderboard.md]:
```suggestion
- Score definition: `F1 instance strict`
```

**Review by tschaffter — CHANGES_REQUESTED** [review]

**Pull request merged**